CONTRIBUTING.md

```diff
@@ -586,33 +586,6 @@ or that are not particularly interesting.
 Here are a few bits and pieces that are referenced from some of the
 scenarios in this guide.
 
-### Pull Request Guidelines
```

Comment:
I was thinking about that before sending the pull request. I agree with you. Would you like me to add it back in? Ideally it should contain a link to the PR guidelines once they're up.